scripts/variantstore/wdl/extract/test_create_variant_annotation_table.py

```diff
@@ -8,37 +8,105 @@
-        with open(dir + 'vat_expected_likely_benign.json') as vat_expected_likely_benign, open(dir + 'vat_test_likely_benign.json') as vat_test_likely_benign:
-            likelyBenignExpectedVAT = json.load(vat_expected_likely_benign)
-            likelyBenignNirvanaOutput = json.load(vat_test_likely_benign)
-            actual = make_annotated_json_row(
-                row_position=5226567,
-                row_ref="A",
-                row_alt="C",
-                variant_line=likelyBenignNirvanaOutput,
-                transcript_line=None)
-            expected = likelyBenignExpectedVAT
-            self.maxDiff=None
-            self.assertEqual(actual, expected)
+        actual = make_annotated_json_row(
+            row_position=5226567,
+            row_ref="A",
+            row_alt="C",
+            variant_line=likelyBenignNirvanaOutput,
+            transcript_line=None)
+        expected = likelyBenignExpectedVAT
+        self.maxDiff=None
+        self.assertIsNone(actual.get('clinvar_id')) # This should not exist---there are no Clinvar values with matching alleles
+        self.assertIsNone(actual.get('clinvar_phenotype')) # No Clinvar values with matching alleles
+        self.assertIsNone(actual.get('clinvar_classification')) # No Clinvar values with matching alleles
+        self.assertEqual(actual, expected)
+
+
+    def test_clinvar_ordering(self):
+        clinvar_swap = [{ 'id': 'RCV01', \
+                      'significance': [ 'sleater', 'kinney', 'guitar', 'solo', 'pathogenic'], \
+                      'refAllele': 'TCT', \
+                      'altAllele': '-', \
+                      'phenotypes': ['brunette'],
+                      'lastUpdatedDate': '2020-03-01'}, \
+                      { 'id': 'RCV02', \
+                      'significance': [ 'pathogenic', 'LikELy paTHoGenIc', 'conflicting data from submitters'], \
+                      'refAllele': 'TCT', \
+                      'altAllele': '-', \
+                      'phenotypes': ['blonde'],
+                      'lastUpdatedDate': '2020-03-02'}]
+        clinvarOrderingNirvanaOutput = copy.deepcopy(cysticFibrosisNirvanaOutput)
+        clinvarOrderingNirvanaOutput["clinvar"] = clinvar_swap
+        actual = make_annotated_json_row(
+            row_position=117559590,
+            row_ref="ATCT",
+            row_alt="A",
+            variant_line=clinvarOrderingNirvanaOutput,
+            transcript_line=None)
+        self.maxDiff=None
+        self.assertEqual(actual.get('clinvar_classification'), ['likely pathogenic', 'pathogenic', 'conflicting data from submitters', 'guitar', 'kinney', 'sleater', 'solo'])
+        self.assertEqual(actual.get('clinvar_id'), ['RCV01', 'RCV02'])
+        self.assertEqual(actual.get('clinvar_phenotype'), ['blonde', 'brunette'])
+        self.assertEqual(actual.get('clinvar_last_updated'), '2020-03-02')
+
 
+    def test_clinvar_inclusion(self):
+        clinvar_swap = [{ 'id': 'RCV01', \
```

Comment:
Do you want to add a test case for a case where the id does NOT start with 'RCV'?

Reply:
Yes and I think that line 88 should handle that case